Mathlib/LinearAlgebra/Multilinear/Curry.lean

```diff
@@ -223,87 +223,114 @@ def multilinearCurryRightEquiv :
 
 namespace MultilinearMap
 
-variable {ι' : Type*} {R M₂}
 
-/-- A multilinear map on `∀ i : ι ⊕ ι', M'` defines a multilinear map on `∀ i : ι, M'`
-taking values in the space of multilinear maps on `∀ i : ι', M'`. -/
-def currySum (f : MultilinearMap R (fun _ : ι ⊕ ι' => M') M₂) :
-    MultilinearMap R (fun _ : ι => M') (MultilinearMap R (fun _ : ι' => M') M₂) where
+variable {R M₂} {ι' : Type*} {N : (ι ⊕ ι') → Type*}
```

Comment:
Can you hoist `ι'` to the top of the file so that it appears just after `ι` in the argument list?